content/docs/04-clusters/03-edge/04-edgeforge-workflow/12-build-artifacts.md

```diff
@@ -0,0 +1,309 @@
+<br /> 
+
+  ```bash
+  sudo mv palette-edge /bin/
+  ```
+
+
+4. Verify the installation of the Palette Edge CLI by issuing the following command. The output will display information about the installed version of the Palette Edge CLI. 
+<br />
+
+  ```bash
+  palette-edge show
+  ```
+
+  ```bash hideClipboard
+  # Sample output   
+  ┌────────────────────────────────────────────────────────────────────────┐
+  | OS Flavor     | Description        | Base Image URI                    |
+  | opensuse-leap | Opensuse Leap 15.4 | quay.io/kairos/core-opensuse-leap |
+  | ubuntu-20     | Ubuntu 20.4 LTS    | quay.io/kairos/core-ubuntu-20-lts |
+  | ubuntu-22     | Ubuntu 22.4 LTS    | quay.io/kairos/core-ubuntu-22-lts |
+  └────────────────────────────────────────────────────────────────────────┘
+  ┌─────────────────────────────────────────────────────────────────────────────────────────────┐
+  | K8S Flavor | Description        | Supported Versions                                        |
+  | k3s        | Rancher K3s        | 1.25.2-k3s1,1.24.6-k3s1,1.23.12-k3s1,1.22.15-k3s1         |
+  | kubeadm    | Kubernetes kubeadm | 1.25.2,1.24.6,1.23.12,1.22.15                             |
+  | rke2       | Rancher RK2        | 1.25.2-rke2r1,1.24.6-rke2r1,1.23.12-rke2r1,1.22.15-rke2r1 |
+  └─────────────────────────────────────────────────────────────────────────────────────────────┘
+  ┌─────────────────────────────────┐
+  | Component             | Version |
+  | Spectro Agent Version | v3.4.3  |
+  | Kairos Version        | v2.0.3  |
+  └─────────────────────────────────┘
+  ```
+<br />
+
+
+5. The next few steps will guide you to create a cluster profile referencing the provider image you must have already pushed to an image registry. Check the **Prerequisites** section if you do not have a provider image in an image registry. 
+
+  Open a web browser and log in to [Palette](https://console.spectrocloud.com). Ensure you are in the **Default** project scope before creating a cluster profile. 
+
+
+6. Navigate to the left **Main Menu** and select **Profiles**. Click on the **Add Cluster Profile** button, and fill out the required basic information fields to create a cluster profile for Edge. 
+
+
+7. Add the following OS layer in the **Profile Layers** section.
+
+  |**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+  |---|---|---|---|
+  |OS|Public Repo|BYOS Edge OS|`1.0.0`|
+
+
+8. Replace the OS layer manifest with the following custom manifest so that the cluster profile can pull the provider image from the ttl.sh image registry. This manifest is the output CanvOS returned after building the Edge artifacts in the [Build Edge Artifacts](/clusters/edge/edgeforge-workflow/palette-canvos) workflow.
```

Comment:
If the Build Edge Artifacts guide is a prerequisite, why not reference the cluster profile they created when completing the guide? That would reduce several steps from this guide. The only thing we need is the cluster profile ID. 

We can essentially remove steps 7 - 12.